packages/base-driver/lib/protocol/protocol.js

```diff
@@ -19,6 +19,7 @@ import { generateDriverLogPrefix } from '../basedriver/helpers';
 export const CREATE_SESSION_COMMAND = 'createSession';
 export const DELETE_SESSION_COMMAND = 'deleteSession';
 export const GET_STATUS_COMMAND = 'getStatus';
+export const GET_APPIUM_SESSION_CAPABILITIES = 'getAppiumSessionCapabilities';
```

Comment:
COMMAND suffix is missing

Reply:
https://github.com/appium/appium/pull/20936/commits/12a1cb3123d72333fee7725ded098fc77cb07737